utils/author_catalog.py

```diff
@@ -54,7 +67,7 @@ def create_catalog(filepath: str = catalog_file_path) -> Dict:
 
     Args:
         filepath (str): The first argument. The filepath of the source catalog of the related database.
-
+        log (bool):
```

Comment:
could you explain what this means? bool generally start with `is_`, by the way.

Reply:
this variable asks the user do you want to create "log file" or not.

Reply:
What is the type of True/False? when I'm looking the type() it says bool.